2023/did-wg.html

```diff
@@ -360,9 +343,6 @@ <h2>Success Criteria</h2>
     TAG <a href="https://www.w3.org/TR/design-principles/">Web Platform Design Principles</a>.
     </p>
 	
-    <!-- Relevance and momentum -->
-	  <p>All new features should be supported by at least two intents to implement before being incorporated in the specification.</p>
```

Comment:
I don't like how this moved here: https://github.com/w3c/charter-drafts/pull/448/files#r1343224378

Reply:
This wording came directly from the charter template, from which it will probably be removed soon (see https://github.com/w3c/charter-drafts/pull/437). The notion of "intent to implement" is quite browser specific, and therefore does not apply here.

The other changes in this commit (esp. [line 319](https://github.com/w3c/charter-drafts/pull/448/files#diff-c5c4868daab54b6181b6e7d7cc17c914e9678f46e80cded37f7c3cbde7b504d5R319)) are meant to cover the same general idea, while being more adapted and more specific to this WG.

Reply:
done: https://github.com/w3c/charter-drafts/pull/448/files#r1350767221

Reply:
> done: https://github.com/w3c/charter-drafts/pull/448/files#r1350767221

Sorry, I don't understand this comment... and the anchor does not seem to link to any specific place in the file...